components/brave_wallet/browser/zcash/zcash_create_transparent_to_orchard_transaction_task.cc

```diff
@@ -75,7 +89,8 @@ bool ZCashCreateShieldTransactionTask::CreateTransaction() {
       2 /* actions count for 1 orchard output no orchard inputs */);
   if (!pick_transparent_inputs_result) {
     error_ = l10n_util::GetStringUTF8(IDS_WALLET_INTERNAL_ERROR);
-    return false;
+    ScheduleWorkOnTask();
+    return;
   }
```

Comment:
Out of scope of this PR but we should think for BTC and ZEC if such pattern should be everywhere replaced with 
```
FailWithError(...);
return;
```